tests/rhtap/types.go

```diff
@@ -0,0 +1,48 @@
+package rhtap
+
+import (
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Remotes struct {
+	Origin string `yaml:"origin"`
+}
+
+type Git struct {
+	Remotes Remotes `yaml:"remotes"`
+}
+
+type Samples struct {
+	Name        string `yaml:"name"`
+	DisplayName string `yaml:"displayName"`
+	Language    string `yaml:"language"`
+	ProjectType string `yaml:"projectType"`
+	Git         Git    `yaml:"git"`
+}
+
+type Stacks struct {
+	SchemaVersion string    `yaml:"schemaVersion"`
+	Samples       []Samples `yaml:"samples"`
```

Comment:
I think `ExtraDevfileEntries` would be the best descriptor for this entity.

```suggestion
type ExtraDevfileEntries struct {
	SchemaVersion string    `yaml:"schemaVersion"`
	Samples       []SampleEntry `yaml:"samples"`
```